src/routes/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import Test from '../pages/Test';
+import News from '../pages/news/News';
+
+const AppRoutes = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Test />} />
```

Comment:
La tarea dice:
_Al ingresar a la ruta base “/“, mostrar un componente_

Generalmente la ruta / es para el home. En vez de crear un componente Test que luego va a ser eliminado, aprovecharía y crearía el componente Home, para que luego pueda ser reutilizado para los cambios

Reply:
Create the page folder and and index
![imagen](https://user-images.githubusercontent.com/75862846/159788905-5f73f481-5b2a-48a0-8ddd-fcb682d4382a.png)

**refactor**: 
import the home page
![imagen](https://user-images.githubusercontent.com/75862846/159789515-540154f5-d2fb-49e6-a303-72e9552aa32d.png)

and change the name of the route 
![imagen](https://user-images.githubusercontent.com/75862846/159789250-caad7b5a-0df3-4628-b7ce-618916c2c2b0.png)